<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Air Sample Gamma Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tab_wid">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="menu_opcnf"/>
   </widget>
   <widget class="QMenu" name="menuCal">
    <property name="title">
     <string>Calibraciones</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Análisis</string>
    </property>
    <addaction name="menu_pks"/>
    <addaction name="menu_cua_air"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCal"/>
   <addaction name="menuAnalysis"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QToolBar" name="toolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="action_zoom"/>
   <addaction name="action_log"/>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>Open1</string>
   </property>
  </action>
  <action name="menu_opcnf">
   <property name="text">
    <string>Open CNF File...</string>
   </property>
  </action>
  <action name="action_log">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
   <property name="toolTip">
    <string>No se que poner aqui</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_zoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="menu_pks">
   <property name="text">
    <string>Búsqueda de Fotopicos...</string>
   </property>
  </action>
  <action name="menu_cua_air">
   <property name="text">
    <string>Cuantificación de Fotopicos (Aire)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
